### ISSUE TITLE
✨ Add `rgba()` color into `config.colors`

### ISSUE THREAD

**Issue by creart-jetsai** [master-co/css#105]
### Description

```js
// master.css.js

export default {
    colors: {
        primary: '#22a3e5',  // works
        ['accordin-expanded']: 'primary/.4', // not works
        ['accordin-expanded']: 'rgba(0,0,0,0.4)' // not works
    }
}
```

We should allow config colors include rgba values and opacity shorthand.

**Comment by 1aron** [comment]
We might implement the following syntax:
```ts
export default {
    colors: {
        primary: '#00000', // HEX
        primary: 'rgb(0,0,0)', // RGB
        primary: 'rgb(0 0 0)', // RGB
        primary: 'rgb(0 0 0/.5)', // RGB
        primary: 'rgba(0,0,0,.5)', // RGBA
     }
}
```

**Comment by itsezc** [comment]
> We might implement the following syntax:
> 
> ```ts
> export default {
>     colors: {
>         primary: '#000', // HEX
>         primary: 'rgb(0,0,0)', // RGB
>         primary: 'rgb(0 0 0)', // RGB
>         primary: 'rgb(0 0 0/.5)', // RGB
>         primary: 'rgba(0,0,0,.5)', // RGBA
>         primary: 'blue/.5'
>      }
> }
> ```

Is `blue/.5` a reference to `colors.blue` ?

**Comment by mockingjet** [comment]
Is `gradient` also supported?

I think it mostly applies on `background` property, but generally it represents a **color** as well.

Or we should stick to use `gradient` only in `config.values`.

**Comment by 1aron** [comment]
> > We might implement the following syntax:
> > ```ts
> > export default {
> >     colors: {
> >         primary: '#000', // HEX
> >         primary: 'rgb(0,0,0)', // RGB
> >         primary: 'rgb(0 0 0)', // RGB
> >         primary: 'rgb(0 0 0/.5)', // RGB
> >         primary: 'rgba(0,0,0,.5)', // RGBA
> >         primary: 'blue/.5'
> >      }
> > }
> > ```
> 
> Is `blue/.5` a reference to `colors.blue` ?

Yup

**Comment by 1aron** [comment]
> Is `gradient` also supported?
> 
> I think it mostly applies on `background` property, but generally it represents a **color** as well.
> 
> Or we should stick to use `gradient` only in `config.values`.

We can't support it in `config.colors` because it contains multiple colors and behaves completely differently.
You are right, we should stick to use gradient in `config.values` or `config.classes` to keep color configuration simple.

**Comment by 1aron** [comment]
🎉 The feature has been released in [v2.0.0-beta.102](https://github.com/master-co/css/releases/tag/v2.0.0-beta.102)
